src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -3519,7 +3522,7 @@ namespace PackagedTerminalHeatPump {
 			PTUnit( PTUnitNum ).LastMode = HeatingMode;
 
 			//   set fluid-side hardware limits
-			if ( PTUnit( PTUnitNum ).HeatCoilFluidInletNode > 0 ) {
+			if ( PTUnit( PTUnitNum ).HeatCoilFluidInletNode > 0 && PTUnit( PTUnitNum ).UnitType_Num != PTWSHPUnit ) {
```

Comment:
This is what caused the crash in release mode. The WSHP water inlet node was not available and caused a crash when InitComponentNodes was called (I changed the name of this variable and this popped up). I think maybe that InitComponentNodes should also be called for WSHP coils but will address that later.